### PR TITLE
Make ignore main optional

### PR DESCRIPTION
**Pull request by jvm986**
Addresses https://github.com/PanagiotisPtr/cov-diff/issues/10

### CODE REVIEW

**Comment by PanagiotisPtr** [comment]
That integration test fail is OK. It's just failing to publish the comment - the build passed. The github action for publishing comments doesn't work on PRs from forks: https://github.com/mshick/add-pr-comment#proxy-for-fork-based-prs

Let me know if you're happy with the PR and I'll merge

File: action.yaml [hunk 1]
```diff
@@ -11,6 +11,8 @@ inputs:
     description: "path to diff file"
   module:
     description: "the Go module name"
+  ignore-main:
+    description: "ignore main package"
```
Comment (PanagiotisPtr) [inline action.yaml:15]: I think you can also add `default: "true"` here since we are ignoring main by default

Comment (jvm986) [inline action.yaml:15]:
```suggestion
    default: "true"
    description: "ignore main package"
```

**Comment by jvm986** [comment]
Apologies, I have been sick. I addressed the comment and I think it would be ready to merge.

Thanks again!

**Comment by PanagiotisPtr** [comment]
No worries, thanks for the contribution. Looks good. I'll merge it in and if there's anything I'll push a subsequent fix (should be ok though)

**Pull request merged**